### PR TITLE
Alfajores changes & comment on unlocking accounts

### PR DESCRIPTION
**Pull request by tkporter**
### Description

This includes the changes to docs & the cli made in the `alfajores` branch to get things working. I added a comment about unlocking every time a node is restarted. I tried running the command to start the node with `--unlock $ACCOUNT` and ran into the same issues with reading input in a docker container we've seen before:
```
Unlocking account dba1ebb0cea8a91bdd9e46d8361ac4c236e6e7e7 | Attempt 1/3
Fatal: Failed to read passphrase: liner: function not supported in this terminal
```
so I figured it'd be better to not have to deal with that bad UX.

I'm going to create a PR to the `alfajores` branch with the unlocking comment as well

### Tested

n/a

### Other changes

n/a

### Related issues

n/a

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/cli/src/commands/lockedgold/lockup.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default class Commitment extends BaseCommand {
     const noticePeriod = new BigNumber(res.flags.noticePeriod)
     const goldAmount = new BigNumber(res.flags.goldAmount)
 
-    if (!(await lockedGold.isVoting(address))) {
+    if (await lockedGold.isVoting(address)) {
```
Comment (kevjue) [inline packages/cli/src/commands/lockedgold/lockup.ts]: Seems like the  `failWith` parameter needs to be updated.  E.g. to `require(isVoting(address)) => false`

File: packages/cli/package.json [hunk 2]
```diff
@@ -29,19 +29,19 @@
     "test": "TZ=UTC jest"
   },
   "dependencies": {
+    "@celo/contractkit": "^0.1.6",
```
Comment (mcortesi) [inline packages/cli/package.json:32]: How recent is this version? is this == as master? or was it a while ago?

There's a small change that makes master != to alfajores: GasPriceMinimun has `proposerFraction` and before was `infrastructureFraction`

Comment (tkporter) [reply packages/cli/package.json:32]: This is the version of contractkit I had to redeploy in order to get the CLI to work on Alfajores-- I think it was around October 3/4. Is it safe to have this contractkit version?

Comment (mcortesi) [reply packages/cli/package.json:32]: i believe so

**Review by mcortesi — APPROVED** [review]

**Pull request merged**